src/webvr-polyfill.js

```diff
@@ -54,6 +54,11 @@ WebVRPolyfill.prototype.isDeprecatedWebVRAvailable = function() {
   return ('getVRDevices' in navigator) || ('mozGetVRDevices' in navigator);
 };
 
+WebVRPolyfill.prototype.connectDisplay = function(vrDisplay) {
+  vrDisplay.fireVRDisplayConnect_();
```

Comment:
That's my understanding as well, but this handled from the perspective of these shim displays don't exist/aren't connected until they're created. Also probably low usage using these events in polyfill for that very reason (these can't be "connected"/"disconnected" in the middle of a session)